feature-libs/product/configuration/src/textfield-configurator/facade/configurator-textfield.service.ts

```diff
@@ -66,32 +73,48 @@ export class ConfiguratorTextfieldService {
    * Adds the textfield configuration to the cart, specified by its product code.
    *
    * @param productCode - Product code
+   * @param configuration Textfield configuration
    */
-  public addToCart(productCode: string) {
+  addToCart(
```

Comment:
Please specify the return type.
And I'd be happy to discuss how to refactor this one, as we generally avoid subscribing unless it's absolutely necessary.
Calling this method multiple times will result in memory leaks (as subscriptions in not garbage collected)

Reply:
I think we took over this call from somewhere in the standard cart handling. But yes, let's discuss..I will schedule a call(wanted to ask you some general questions anyhow)